pkg/rdkafka/JsonSerializer.php

```diff
@@ -6,34 +6,30 @@
 
 class JsonSerializer implements Serializer
 {
-    public function toString(RdKafkaMessage $message): string
+    public function toString(RdKafkaMessage $message): ?string
```

Comment:
That's not true, and it does not match with the `Stringable` interface that is part of PHP8.

See https://www.php.net/manual/en/class.stringable.php.

```suggestion
    public function toString(RdKafkaMessage $message): string
```

Reply:
But `Serializer` does not extend `Stringable`?

Reply:
And [Stringable](https://www.php.net/manual/en/class.stringable.php) defines interface `public __toString(): string` method with two underscores in name, not `toString`